config/jest/test.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+    rootDir: "../../",
+    transform: {
+        "^.+\\.jsx?$": "<rootDir>/config/jest/test.transform.js",
+    },
+    testEnvironment: "jest-environment-jsdom",
+    globals: {
+        SNAPSHOT_INLINE_APHRODITE: true,
+    },
+    testMatch: [
+        "<rootDir>/packages/**/*.test.js",
+        "<rootDir>/shared-unpackaged/*.test.js",
```

Comment:
This is good, but does it pass the test of "least surprise"? Like if we added tests for some of our tooling or node scripts, would we expect them to just run? I am on the fence about it, but I did prefer when we were guaranteed to pick up all tests in the entire repo rather than each location be opt-in.

Reply:
I can switch it back to a regex or we can do `"<rootDir>/**/*.test.js"` and I can change the `vite` tests to have a different pattern.

Reply:
I think we probably should change it. I am all for the vite tests having a different pattern anyway because I am in love with my `.vist.js` suggestion. 😂